### PR TITLE
updating Cloud messaging details

### PR DESCRIPTION
**Pull request by flossypurse**
- Removes link to Temporal Cloud address
- Adds explicit message to every top level Cloud page that Temporal Cloud is accessible by invitation only - but can join waitlist
- Bumps Temporal Cloud up 1 on the sidebar

### CODE REVIEW

File: docs/cloud-context/namespaces-create.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 ### Create a Namespace using Temporal Cloud UI
 
 1. Gather the information listed earlier in [Information needed to create a Namespace](#information-needed-to-create-a-namespace).
-1. In a browser, go to [https://cloud.temporal.io](https://cloud.temporal.io) and log in.
+1. Go to to Temporal Cloud UI and log in.
```
Comment (djmagee) [inline docs/cloud-context/namespaces-create.md:32]: De-dupe "to to". (Doing reviews on a phone sucks.)

Comment (flossypurse) [inline docs/cloud-context/namespaces-create.md:32]:
```suggestion
1. Go to the Temporal Cloud UI and log in.
```

Comment (flossypurse) [reply docs/cloud-context/namespaces-create.md:32]: Good catch!

File: docs/cloud-context/namespaces-create.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 ### Create a Namespace using Temporal Cloud UI
 
 1. Gather the information listed earlier in [Information needed to create a Namespace](#information-needed-to-create-a-namespace).
-1. In a browser, go to [https://cloud.temporal.io](https://cloud.temporal.io) and log in.
+1. Go to the Temporal Cloud UI and log in.
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs/cloud-context/namespaces-create.md:32]: We are purposefully not linking to it atm.

**Pull request merged**